sys-auth/sssd/sssd-2.3.1.ebuild

```diff
@@ -12,32 +14,35 @@ KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sparc ~x
 
 LICENSE="GPL-3"
 SLOT="0"
-IUSE="acl +autofs +locator +netlink nfsv4 nls +manpages samba selinux +sudo +ssh systemd test valgrind"
+IUSE="acl +autofs doc kcm +locator +netlink nfsv4 nls +manpages pac python samba secrets selinux sudo ssh systemd test valgrind"
```

Comment:
We have quite a lot of USE flags now, which is fine, but we don't always need to expose them jsut because an option exists. Are they all valuable?

Reply:
i *guess* pac can go since it doesn't add another dependency but im not so sure about the other ones...

Reply:
When I discovered that USE="ssh sudo" was required to build the tests, I thought about removing the flags entirely.

They, along with IUSE=autofs, seem like very low value options. They each install something like two extra files and have no additional dependencies.

Given the, seemingly inherent, complexity of this package I think removing USE flags like these is probably a good idea.

Reply:
i have gone ahead and yanked autofs and ssh. removed kcm and secrets to merge with systemd flag

sudo needs changes to app-admin/sudo

Reply:
Can you elaborate on what you mean about needing changes to `app-admin/sudo`?

Would be good to sort that out, because I think sssd's tests need sudo support enabled.

Reply:
sssd use on sudo itself need to turn into sssd(+) or whatever the correct syntax was.

Reply:
In other words, app-admin/sudo needs to have USE=sssd enabled by default? How could that help?

Maybe let's back up a step, and tell me what breaks if we remove IUSE=sudo from `sys-auth/sssd` and enable the option always?

Reply:
sorry, no, i meant something else

```sssd? ( sys-auth/sssd[sudo] )```

needs to be ```sssd? ( sys-auth/sssd[sudo(+)] )```

i would imagine pkgcore yelling if we would just remove sudo use from sssd without doing that first.

and i also just realized i forgot to push something, pac doesn't *really* need to be a own useflag, it could be condensed into samba too.
but i guess its not a huge deal.